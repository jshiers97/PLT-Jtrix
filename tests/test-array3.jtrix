void main() {
	float[] a;
	int i;

	a = [0.,0.,0.,0.,0.,0.,0.,0.,0.,0.];
	a[0] = 1.; 
	for(i = 1; i < 10; i = i++) {
		a[i] = a[i-1] * int_to_f(i);
	}

	for(i = 0; i < 10; i = i++) {
		printf(a[i]); 
	}

	free(a);
}
