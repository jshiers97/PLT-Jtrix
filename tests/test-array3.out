1.000
1.000
2.000
6.000
24.000
120.000
720.000
5040.000
40320.000
362880.000
